Bio/PDB/__init__.py

```diff
@@ -88,3 +97,67 @@
     from .SASA import ShrakeRupley
 except ImportError:
     pass
+
+_FormatToParser = {"pdb": PDBParser(), "cif": MMCIFParser()}
+
+
+def read(handle, format, id=None):
+    """Parse a file or handle using one of the available parsers to generate a structure.
+
+    Arguments:
+     - handle    - handle to the file, or the filename as a string
+     - format    - string describing the file format. can be [pdb, cif, mmtf]
```

Comment:
I would suggest to use `fmt` instead of `format`, as `format` is also a built-in function. Over time we will probably start using `fmt` instead of `format` for existing parsers like `Bio.SeqIO`; some parsers in Biopython are already using `fmt`, for example `Bio.motifs`.

Reply:
happy to change this. I went with the same style as `SeqIO.read`. but if other parsers already use fmt I think I would be smart to also use `fmt` here if that is the long term goal.

Reply:
Python tolerates using "format" as an argument or variable name, but I agree using "fmt" is better on new code.

Reply:
Agree with `fmt`. I'd go ahead and add a kwargs that you can pass to the parser. Something like this:

```python
def read(handle, fmt, id=None, **kwargs):
    ....
    parser = Parser(**kwargs)
    ...
```

That way you can write `read(myhandle, fmt="pdb", QUIET=1)` and that `QUIET` keyword gets passed to the parsers.